templates/DIF/email/data-managers/data-managers.dif-submitted.email.twig

```diff
@@ -3,23 +3,23 @@
 {% block body_html %}
 Dear {{ recipient.firstName }} {{ recipient.lastName }},<br>
 <br>
-A Dataset Information Form (DIF) has been saved and submitted to the Gulf of Mexico Research Initiative Information and Data Cooperative (GRIIDC) by {{ user.firstName }} {{ user.lastName }} for dataset {{ dif.dataset.udi }}. {{ user.firstName }} {{ user.lastName }} is a member of {{ user.getResearchGroupNames() | join(', ') }} and the DIF submitted is associated with {{ dif.researchGroup.name }}.<br>
+A Dataset Information Form (DIF) has been saved and submitted to the GRIIDC by {{ user.firstName }} {{ user.lastName }} for dataset {{ dif.dataset.udi }}. {{ user.firstName }} {{ user.lastName }} is a member of {{ user.getResearchGroupNames() | join(', ') }} and the DIF submitted is associated with {{ dif.researchGroup.name }}.<br>
 <br>
 Thank you,<br>
 <br>
 The GRIIDC Team<br>
 <br>
-*You are receiving this email because you are identified as a data manager for a Gulf of Mexico Research Initiative Research Consortium. If you believe you should not be receiving this email please notify <a href="mailto:griidc@gomri.org">griidc@gomri.org</a>.*
+*You are receiving this email because you are identified as a data manager for this research group. If you believe you should not be receiving this email please notify <a href="mailto:griidc@gomri.org">griidc@gomri.org</a>.*
 {% endblock %}
 
 {% block body_text %}
 Dear {{ recipient.firstName }} {{ recipient.lastName }},
 
-A Dataset Information Form (DIF) has been saved and submitted to the Gulf of Mexico Research Initiative Information and Data Cooperative (GRIIDC) by {{ user.firstName }} {{ user.lastName }} for dataset {{ dif.dataset.udi }}. {{ user.firstName }} {{ user.lastName }} is a member of {{ user.getResearchGroupNames() | join(', ') }} and the DIF submitted is associated with {{ dif.researchGroup.name }}.
+A Dataset Information Form (DIF) has been saved and submitted to the GRIIDC by {{ user.firstName }} {{ user.lastName }} for dataset {{ dif.dataset.udi }}. {{ user.firstName }} {{ user.lastName }} is a member of {{ user.getResearchGroupNames() | join(', ') }} and the DIF submitted is associated with {{ dif.researchGroup.name }}.
 
 Thank you,
 
 The GRIIDC Team
 
-*You are receiving this email because you are identified as a data manager for a Gulf of Mexico Research Initiative Research Consortium. If you believe you should not be receiving this email please notify griidc@gomri.org.*
+*You are receiving this email because you are identified as a data manager for this research group. If you believe you should not be receiving this email please notify griidc@gomri.org.*
```

Comment:
should we try and mention the RG? This may be a bit too generic.

Reply:
Rosalie gave me the wording